src/main/scala/com/kinja/amqp/persistence/DataSources.scala

```diff
@@ -0,0 +1,7 @@
+package com.kinja.amqp.persistence
+
+trait DataSources {
```

Comment:
I don't think this trait is necessary. Shouldn't it be just an abstract value in the MySqlMessageStore, of type javax.sql.DataSource?